url/handlers.go

```diff
@@ -0,0 +1,128 @@
+// // url/handlers.go
```

Comment:
Probably delete this if we aren't using it? I'm a bit confused on the structure of your code